test/public/defaults.js

```diff
@@ -616,16 +611,45 @@ module.exports.expectColumnValues = async (page, columnId, expectedInnerTextValu
     }
 };
 
+/**
+ * Method to check cells of a row with given id have expected innerText
+ *
+ * @param {puppeteer.Page} page the puppeteer page
+ * @param {stirng} rowId row id
+ * @param {Object<string, string>} [expectedInnerTextValues] values expected in the row
+ *
+ * @return {Promise<void>} resolve once row's values were checked
+ */
+module.exports.expectRowValues = async (page, rowId, expectedInnerTextValues) => {
```

Comment:
Where is this used? It looks like it's not, and tbh I think it's a good thing because I don't really see where it make sense.